src/components/Sender/Sender.tsx

```diff
@@ -60,16 +67,15 @@ const Sender = () => {
         onChange={changeTitle}
       />
       <Recipients />
-      <Button
-        variant="contained"
-        color="primary"
-        onClick={sendEmails}
-        // disabled={!recipients.length}
-      >
+      <Button variant="contained" color="primary" onClick={sendEmails}>
         Send
       </Button>
     </Grid>
   );
 };
 
-export default memo(Sender);
+Sender.propTypes = {
+  enqueueSnackbar: PropTypes.func.isRequired,
+};
+
+export default withSnackbar(Sender);
```

Comment:
🐛 This should be memoized

```suggestion
export default memo(withSnackbar(Sender));
```